jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/util/GenericErrorSummaryCountCell.java

```diff
@@ -27,32 +27,35 @@
 import org.jboss.errai.ui.client.local.api.IsElement;
 import org.jboss.errai.ui.shared.api.annotations.DataField;
 import org.jboss.errai.ui.shared.api.annotations.Templated;
-import org.jbpm.workbench.pr.client.resources.i18n.Constants;
-import org.jbpm.workbench.pr.model.ProcessInstanceSummary;
+import org.jbpm.workbench.common.client.list.AbstractMultiGridPresenter;
+import org.jbpm.workbench.common.model.GenericErrorSummary;
+import org.jbpm.workbench.common.client.resources.i18n.Constants;
+
 
 @Dependent
-@Templated(value = "ProcessInstanceSummaryErrorCountCell.html", stylesheet = "ProcessInstanceSummaryErrorCountCell.css")
-public class ProcessInstanceSummaryErrorCountCell extends AbstractCell<ProcessInstanceSummary> implements IsElement {
+@Templated(value = "GenericErrorSummaryCountCell.html", stylesheet = "GenericErrorSummaryCountCell.css")
+public class GenericErrorSummaryCountCell<C extends GenericErrorSummary> extends AbstractCell<C> implements IsElement {
 
     private static final String TOOLTIP_NAME = "pisepc-tooltip";
-    private static final String PROCESS_INSTANCE_ATTRIBUTE = "data-jbpm-processInstanceId";
+    private static final String DATA_INSTANCE_ATTRIBUTE = "data-jbpm-id";
     private static final String ERROR_PRESENT_STYLE = "error-present";
     private static final String LINK_AVAILABLE_STYLE = "link-available";
 
     @Inject
     @DataField(TOOLTIP_NAME)
     private Anchor tooltip;
 
-    private ProcessInstanceListPresenter viewPresenter;
-
-    public ProcessInstanceSummaryErrorCountCell init(final ProcessInstanceListPresenter viewPresenter) {
+    private AbstractMultiGridPresenter viewPresenter;
+    
```

Comment:
just get rid of some extra spacing, there are quite a few places in the PR like this.

Reply:
@cristianonicolai @bsremac All your requested changes are already done :)